packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -1522,17 +1403,17 @@ describe('meeting', () => {
       it('should reject if joinWebUrl is incorrect URL in string format', async () => {
         await utils.initializeWithContext(FrameContexts.content);
         expect(() =>
-          meeting.joinPrivateMeeting({
-            ...mockJoinPrivateMeetingParams,
+          meeting.joinMeeting({
+            ...mockjoinMeetingParams,
             joinWebUrl: 'this is incorrect url in string format',
           }),
         ).toThrowError('this is incorrect url in string format');
       });
 
       it(`should successfully joinMeeting`, async () => {
```

Comment:
This unit test looks like a duplicate of the unit test on line 1394 (`https://learn.microsoft.com/en-us/graph/api/resources/onlinemeetingbase?view=graph-rest-1.0`)